src/types.ts

```diff
@@ -147,6 +147,12 @@ export interface TurboLogger {
   debug: (message: string, ...args: any[]) => void;
 }
 
+export type DataItemOptions = {
```

Comment:
Any reason not to just use the arbundles type here?

Reply:
efe73f8